test/controllers/application_controller_test.rb

```diff
@@ -96,55 +94,210 @@ class ApplicationControllerTest < ActionController::TestCase
 
   test 'check privileges function' do
     login_admin
-    @controller.class.instance_variable_set(:@controller_name, 'users')
+
+    @controller.class.instance_variable_set(:@controller_path, 'users')
+    @controller.instance_variable_set(:@auth_privileges, {
+                                        'administration_security_users' => {
+                                          read: true
+                                        }
+                                      })
```

Comment:
Seba, te parece que lo identemos asi:
```suggestion
    @controller.instance_variable_set(
      :@auth_privileges, {
        'administration_security_users' => {
          read: true
        }
      }
    )
```